tools/changelog/PR_FORMAT.md

```diff
@@ -48,9 +48,13 @@ Will be includede in alpha/beta/rc changelog, excluded from stable.
 ```
 
 ## Possible Sections
-
+<!-- 
+When change this, please keep in mind that it is parsed by [changelog.main.kts], and the PR templates should be updated too:
```

Comment:
change -> changing

Or just leave only "Note that this is parsed by [changelog.main.kts]". No need for so many words IMO.

Reply:
Done